storehaus-core/src/main/scala/com/twitter/storehaus/IterableStore.scala

```diff
@@ -1,36 +1,33 @@
 /*
- * Copyright 2014 Twitter inc.
+ * Copyright 2014 Twitter Inc.
  *
- *    Licensed under the Apache License, Version 2.0 (the "License");
- *    you may not use this file except in compliance with the License.
- *    You may obtain a copy of the License at
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain
+ * a copy of the License at
  *
- *        http://www.apache.org/licenses/LICENSE-2.0
+ *     http://www.apache.org/licenses/LICENSE-2.0
  *
- *    Unless required by applicable law or agreed to in writing, software
- *    distributed under the License is distributed on an "AS IS" BASIS,
- *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *    See the License for the specific language governing permissions and
- *    limitations under the License.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
  */
 
 package com.twitter.storehaus
 
-import com.twitter.util.{ Future, Promise, Return }
+import com.twitter.util.Future
 import com.twitter.concurrent.Spool
-import com.twitter.concurrent.Spool.*::
 
 object IterableStore {
   /** Factory method to create a IterableStore from a Map. */
   def fromMap[K, V](m: Map[K, V]): IterableStore[K, V] = new MapStore(m)
 
   /** Helper method to convert Iterator to Spool. */
   def iteratorToSpool[K, V](it: Iterator[(K, V)]): Future[Spool[(K, V)]] = Future.value {
```

Comment:
while we are here, I think it is binary compatible (though not source) to change to:
```scala
def iteratorToSpool[T](it: Iterator[T]): Future[Spool[T]] = ...
```
Not sure why this forces `(K, V)`